src/vs/base/browser/ui/menu/menu.css

```diff
@@ -124,6 +132,8 @@
 
 .monaco-menu .monaco-action-bar.vertical .action-item {
 	border: 1px solid transparent; /* prevents jumping behaviour on hover or focus */
+	border-left: 0;
```

Comment:
this introduces jumping behavior, please remove

Reply:
The jumping in High Contrast mode?

Reply:
Jumping behavior has been fixed.